function/services/application-mapper/index.js

```diff
@@ -25,6 +25,9 @@ async function mapApplicationDataToOracleObject(data, applicationFormJson, addre
         if (key === 'splitFuneral') {
             applicationFormJson.split_funeral = true;
         }
+        if (key === 'origin') {
```

Comment:
Will this break the downstream db if something other than `W`, or `T` is assigned here?

Even though the template's owner schema strictly only allows `W`, or `T`, is it worth doing a check here?